scripts/install-windows-prereqs.ps1

```diff
@@ -5,8 +5,8 @@
 Param(
     [string]$GitURL = 'https://github.com/git-for-windows/git/releases/download/v2.19.1.windows.1/Git-2.19.1-64-bit.exe',
     [string]$GitHash = '5E11205840937DD4DFA4A2A7943D08DA7443FAA41D92CCC5DAFBB4F82E724793',
-    [string]$OpenSSLURL = 'https://slproweb.com/download/Win64OpenSSL-1_1_1f.exe',
-    [string]$OpenSSLHash = '3571bf0fc253312fb7289a12d0f543e5a63846bc628cf94b0bf25f73cf5fd641',
+    [string]$OpenSSLURL = 'https://slproweb.com/download/Win64OpenSSL-1_1_0L.exe',
```

Comment:
Is this an intentional reversion to a 1.1.0 version instead of 1.1.1?

Side note: A specific version of OpenSSL is also not part of ansible/oe-windows-acc-setup.yml for some reason, in case that's not intentional.